packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -570,4 +572,16 @@ export default class SalesforceCRMService implements CRM {
       return [];
     }
   }
+
+  private async getAccountIdBasedOnEmailDomainOfContacts(email: string) {
+    const conn = await this.conn;
+    const emailDomain = email.split("@")[1];
+
+    const response = await conn.query(
+      `SELECT Id, Email, AccountId FROM Contact WHERE Email LIKE '%@${emailDomain}' AND AccountId != null`
+    );
+
+    const accountId = this.getDominantAccountId(response.records as { AccountId: string }[]);
```

Comment:
Nit: Don't need to set this to `accountId`. Can just return.
```suggestion
    return this.getDominantAccountId(response.records as { AccountId: string }[]);
```